API-Mocks/WordPressMocks/src/main/assets/mocks/mappings/files.json

```diff
@@ -0,0 +1,13 @@
+{
+    "request": {
+        "method": "GET",
+        "urlPattern": "/.*([A-z0-9]*\\.[A-z]{1,6})"
+    },
+    "response": {
+        "status": 200,
+        "bodyFileName": "../__files{{request.url}}",
```

Comment:
This approach assumes the request path and the served file path are the same but this is not a requirement.